specifyweb/businessrules/uniqueness_rules.json

```diff
@@ -69,7 +69,7 @@
     ],
     "CollectionObjectGroupJoin": [
         {
-            "rule": [["parentCOG", "childCO"], []],
+            "rule": [["childCO", "parentCOG"], []],
```

Comment:
Ah, I meant childCO be in the different list. So, it'll be 

`[["childCO"], ["parentCOG"]]`

That way, we've an entire infrastructure to say "cannot add CO because it must be unique to this parent COG"